pkg/server/config.go

```diff
@@ -406,9 +405,6 @@ func NewConfig(ctx context.Context, opts kcpserveroptions.CompletedOptions) (*Co
 		virtualWorkspaceServerProxyTransport = transport
 	}
 
-	// Make sure to set our RequestInfoResolver that is capable of populating a RequestInfo even for /services/... URLs.
-	c.GenericConfig.RequestInfoResolver = requestinfo.NewKCPRequestInfoResolver()
```

Comment:
One places where this was used was in the cache server, where it would prune the cluster anyhow:
https://github.com/kcp-dev/kcp/blob/339e173f0b33d49da1d1d58791a14524e6d11c64/pkg/cache/server/config.go#L169-L194